docs/user/api/v3.rst

```diff
@@ -413,6 +417,10 @@ Project update
             "language": "ja",
             "programming_language": "py",
             "homepage": "https://readthedocs.org/",
+            "tags" : [
+                "new tag",
+                "updated tag"
+            ]
```

Comment:
How does "updated tag" work? It sounds like it updates an existing tag? But isn't it more that this set of tags entirely overwrites any existing tags?

Reply:
Oh my bad! The nomenclature I used was ambiguous.
For `PUT`,  omitting them will not change them, but sending a list of tags will replace the old tags.
Should I change the names in the test files?

Reply:
Thanks for clarifying!

For the tests, I think it seems okay. As long as the tests verifies that the behavior is what is intended.

But for the documentation, it would be nice if it's easy to understand what's going on. You could add it in a slightly creative way:

```
            "tags" : [
                "adding a list here",
                "will replace",
                "all existing tags",
                "with these tags"
            ]
```

But otherwise, I think it's also good if you can find a place in the documentation text to explicitly state this behavior.

Reply:
I have made a few changes.

And the docs looks like this.
![image](https://user-images.githubusercontent.com/72622483/197595653-1519d259-90f6-457c-856a-3d4a17476ebd.png)

Reply:
This looks good, yes. It seems more clear now how this part works.